.github/workflows/create-release-pr.yaml

```
@@ -11,10 +11,35 @@ jobs:
     name: Create Release PR
     runs-on: ubuntu-latest
     steps:
+      - name: Check if a branch is whitelisted and maintained
+        id: branch-check
+        run: |
+          BRANCH_NAME="${GITHUB_REF#refs/heads/}"
```

```suggestion
          BRANCH_NAME="${{ github.ref_name }}"
```
use github variable?